src/Country/CountryInterface.php

```diff
@@ -96,4 +96,21 @@ public function getTelephoneCode();
      * @param string $telephoneCode The telephone code.
      */
     public function setTelephoneCode($telephoneCode);
+
+    /**
+     * Gets the country currency code.
+     *
+     * The currency code is the default currency code for a country.
```

Comment:
Let's replace these two sentences with "Represents the default currency used in the country, if known."
No need to mention ISO, we don't do it in the CurrencyInterface either.